cmd/podman/networks/disconnect.go

```diff
@@ -0,0 +1,45 @@
+package network
+
+import (
+	"github.com/containers/podman/v2/cmd/podman/common"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	networkDisconnectDescription = `Remove container from a network`
+	networkDisconnectCommand     = &cobra.Command{
+		Use:               "disconnect [options] NETWORK CONTAINER",
+		Short:             "network rm",
+		Long:              networkDisconnectDescription,
+		RunE:              networkDisconnect,
+		Example:           `podman network disconnect web secondary`,
+		Args:              cobra.MinimumNArgs(2),
+		ValidArgsFunction: common.AutocompleteNetworks,
+	}
+)
+
+var (
+	networkDisconnectOptions entities.NetworkDisconnectOptions
+)
+
+func networkDisconnectFlags(flags *pflag.FlagSet) {
+	flags.BoolVarP(&networkDisconnectOptions.Force, "force", "f", false, "force removal of container from network")
+}
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: networkDisconnectCommand,
+		Parent:  networkCmd,
+	})
+	flags := networkDisconnectCommand.Flags()
+	networkDisconnectFlags(flags)
+}
+
+func networkDisconnect(cmd *cobra.Command, args []string) error {
+	networkDisconnectOptions.Container = args[1]
```

Comment:
Going to need a valid args function on the command to verify we received exactly 2 args

Reply:
Nevermind, I was looking in the wrong place.

Reply:
that is already handled by cobra.
```
 		Args:              cobra.MinimumNArgs(2),
```

Reply:
although maybe I adjust that to must equal 2

Reply:
Exact args does seem more correct